checks.d/http_json.py

```diff
@@ -0,0 +1,56 @@
+from checks import AgentCheck
+import urllib2
+import json
+
+class HTTPJSONCheck(AgentCheck):
+    def check(self, instance):
+        if 'url' not in instance:
```

Comment:
Could you raise an exception instead ?
This way it would be displayed in the info page.